app/routes/dois/show/edit.js

```diff
@@ -27,6 +28,9 @@ export default Route.extend({
               'Using the Form would update this DOI to the latest schema version.'
```

Comment:
This is an API issue, will be addressed in https://github.com/datacite/datacite/issues/2015.

Reply:
For some reason, I didn't see this.  Setting the schema version of a doi via fabrica is done in the update form, regardless of what changes are being made (i.e. whether the change is a state change or some other mod).  

I see no harm in just setting the default schema version on create since that is what is happening.  I will leave a comment on the https://github.com/datacite/datacite/issues/2015 issue for lupo in case this should be completely up to lupo.  

In the meantime setting it in the create form will eliminate the confusion caused by the flash message.

Reply:
Thank you, yes I agree with setting it in the create form.